<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PyWebEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>191</y>
      <width>162</width>
      <height>174</height>
     </rect>
    </property>
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="createFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveAsFileAction"/>
    <addaction name="exitFileAction"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>Инструменты</string>
    </property>
   </widget>
   <widget class="QMenu" name="pluginsMenu">
    <property name="title">
     <string>Плагины</string>
    </property>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="authSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="insertSettingsAction"/>
    <addaction name="appendSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="viewSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="configSettingsAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Редактор</string>
    </property>
    <addaction name="updateEditAction"/>
    <addaction name="tagEditAction"/>
    <addaction name="fontEditAction"/>
    <addaction name="symEditAction"/>
    <addaction name="underlineEditAction"/>
    <addaction name="italicEditAction"/>
    <addaction name="boldEditAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="toolsMenu"/>
   <addaction name="pluginsMenu"/>
   <addaction name="settingsMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFileAction">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="createFileAction">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsFileAction">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="reloadPluginsAction">
   <property name="text">
    <string>Перезагрузить</string>
   </property>
  </action>
  <action name="loadPluginsAction">
   <property name="text">
    <string>Подключить</string>
   </property>
  </action>
  <action name="insertSettingsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Вставка</string>
   </property>
  </action>
  <action name="appendSettingsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Дополнение</string>
   </property>
  </action>
  <action name="configSettingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="exitFileAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="updateEditAction">
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="tagEditAction">
   <property name="text">
    <string>Вставить тег</string>
   </property>
  </action>
  <action name="authSettingsAction">
   <property name="text">
    <string>Авторизация</string>
   </property>
  </action>
  <action name="fontEditAction">
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="symEditAction">
   <property name="text">
    <string>Символы</string>
   </property>
  </action>
  <action name="viewSettingsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Предпросмотр</string>
   </property>
  </action>
  <action name="underlineEditAction">
   <property name="text">
    <string>Подчеркивание</string>
   </property>
  </action>
  <action name="italicEditAction">
   <property name="text">
    <string>Курсив</string>
   </property>
  </action>
  <action name="boldEditAction">
   <property name="text">
    <string>Жирный</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
